lib/statsig/configs.ex

```diff
@@ -99,4 +107,32 @@ defmodule Statsig.Configs do
   end
 
   defp api_client, do: Application.get_env(:statsig, :api_client, Statsig.APIClient)
+
+  defp init_from_bootstrap() do
+    case Application.get_env(:statsig, :bootstrap_config_specs) do
+      nil ->
+        :skip
+      specs when is_map(specs) ->
+        time = update_specs(specs)
+        {:ok, %{State.new() | last_sync_time: time}}
+      other ->
+        Logger.warning("Invalid bootstrap_config_specs format, falling back to network initialization")
+        :skip
+    end
+  end
+
+  defp update_specs(specs) do
```

Comment:
i would expect this to return the updated configs and not just the time, which doesn't seem to be added to the configs that have been updated.